content/user-guide/raspberry.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Raspberry guide
+description: How to run Mysterium dVPN on Raspberry Pi 
+---
+
+## Spin-off Raspberry Pi node in consumer mode
+
+1.  SSH into your RaspberryPi and open editor of /lib/systemd/system/mysterium-node.service file.
+2.  Replace the following line:
+
+```bash
+ExecStart=/usr/bin/myst $CONF_DIR $SCRIPT_DIR $DATA_DIR $RUN_DIR $DAEMON_OPTS service --agreed-terms-and-conditions $SERVICE_OPTS`
```

Comment:
We have a service for consumer mode now:
```
systemctl disable mysterium-node.service
systemctl stop mysterium-node.service

systemctl enable mysterium-consumer.service
systemctl start mysterium-consumer.service
```

Reply:
In that case this should pretty much just point to linux docs and thats that, because it already goes over that.